js/plugins/ollama/src/index.ts

```diff
@@ -59,6 +59,40 @@ export function ollama(params: OllamaPluginParams): GenkitPlugin {
   });
 }
 
+/**
+ * Please refer to: https://github.com/ollama/ollama/blob/main/docs/modelfile.md
+ * for further information.
+ */
+export const OllamaConfigSchema = GenerationCommonConfigSchema.extend({
+  temperature: z
+    .number()
+    .min(0.0)
+    .max(1.0)
+    .default(0.8)
+    .describe(
+      'Controls the degree of randomness in token selection. A lower value is ' +
+        'good for a more predictable response. A higher value leads to more ' +
+        'diverse or unexpected results.'
+    )
+    .optional(),
+  topK: z
+    .number()
+    .min(1)
+    .max(40)
+    .describe('The maximum number of tokens to consider when sampling.')
+    .default(40),
+  topP: z
+    .number()
+    .min(0)
+    .max(1.0)
+    .describe(
+      'Decides how many possible words to consider. A higher value means ' +
+        'that the model looks at more possible words, even the less likely ' +
+        'ones, which makes the generated text more diverse.'
+    )
+    .default(0.9),
```

Comment:
```suggestion
    .max(1.0)
    .default(0.9)
    .describe(
      'Decides how many possible words to consider. A higher value means ' +
        'that the model looks at more possible words, even the less likely ' +
        'ones, which makes the generated text more diverse.'
    ),
```

nit: consistency in the ordering